opm/autodiff/GeoProps.hpp

```diff
@@ -228,7 +229,7 @@ namespace Opm
         /// grid the whole TRAN keyword is quite meaningless.
 
         template <class Grid>
-        const std::vector<data::CellData> simProps( const Grid& grid ) const {
+        const CellDataContainer simProps( const Grid& grid ) const {
```

Comment:
Regardless of name and type, there's no reason for this to be returned as `const`.